### ISSUE TITLE
White space recognition is great before finetuning, and worse after

### ISSUE THREAD

**Issue by inesriahi** [PaddlePaddle/PaddleOCR#8917]
请提供下述完整信息以便快速定位问题/Please provide the following information to quickly locate the problem

- 系统环境/System Environment：Colab
- 版本号/Version：Paddle：  PaddleOCR：PPocr-v3

This issue has been raised previously but without a solution. 
I have a set of documents that I want to extract the text from. Using the model provided for recognition, the reults are good, and the spaces are detected well.

However, after finetuning this recognition model: `en_PP-OCRv3_rec_train`, the accuracy increased significantly from 0.70 to 0.967 on the test data, but when using this model to recognize a new test image, I get many text boxes recognized corrrectly but with removed spaces, although there are others that still have spaces. The annotated data has spaces as this shows:
![image](https://user-images.githubusercontent.com/38370333/215313507-1d6e597b-908f-406d-9026-cf3eca23f791.png)

In the configuration file, I have `use_space_char: true`. I have also tried switching `use_dilation` and trying different values for `det_db_unclip_ratio` and `unclip_ratio`. Trying those has improved the result for some segments but made it worse for others.

One questions I am thinking of though, the `en_dict.txt` has a white space character already as it looks here:
![image](https://user-images.githubusercontent.com/38370333/215313351-70413150-5407-40d6-995b-2c02194bc054.png)
So, why do we have `use_space_char`?

This is an example for the problem after finetuning:
![image](https://user-images.githubusercontent.com/38370333/215312977-d4fcef8f-c878-4b8f-9e72-ce4416d2af46.png)
![image](https://user-images.githubusercontent.com/38370333/215312999-fc0554ce-a361-4636-be67-c3289eff1d6d.png)

I hope that there is a solution for fixing this by now, although that as I said, the model before finetuning was better in recognizing spaces.

I would appreciate any help.

**Comment by tink2123** [comment]
First, please check whether all the spaces are marked in the training data. Looking at the current experimental results, there is a high probability that it is a data problem.

`en_dict.txt` and configuration files do not need to be modified, and the default is to support space recognition.

>So, why do we have use_space_char?

As you said. you dont have to set `use_space_char`

If there is no problem with the labeling of the training data, you can provide us with a few samples, and we'll try to analyze possible reason.
